src/control/cmd/dmg/pretty/pool.go

```diff
@@ -64,6 +67,38 @@ func PrintPoolQueryResponse(pqr *control.PoolQueryResp, out io.Writer, opts ...P
 	return w.Err
 }
 
+// PrintPoolQueryTargetResponse generates a human-readable representation of the supplied
+// PoolQueryTargetResp struct and writes it to the supplied io.Writer.
+func PrintPoolQueryTargetResponse(pqtr *control.PoolQueryTargetResp, out io.Writer, opts ...PrintConfigOption) error {
+	if pqtr == nil {
+		return errors.Errorf("nil %T", pqtr)
+	}
+	w := txtfmt.NewErrWriter(out)
+
+	// Maintain output compatibility with the `daos pool query-targets` output.
+	for infosIdx := range pqtr.Infos {
+		fmt.Fprintf(w, "Target: type %s, state %s\n", pqtr.Infos[infosIdx].Type, pqtr.Infos[infosIdx].State)
+		if pqtr.Infos[infosIdx].Space != nil {
+			for tierIdx, tierUsage := range pqtr.Infos[infosIdx].Space {
+				var tierName string
+				switch tierIdx {
+				case int(control.StorageMediaTypeScm):
+					tierName = fmt.Sprintf("- Storage tier %d (SCM):", tierIdx)
+				case int(control.StorageMediaTypeNvme):
+					tierName = fmt.Sprintf("- Storage tier %d (NVMe):", tierIdx)
+				default:
+					tierName = fmt.Sprintf("- Storage tier %d (unknown):", tierIdx)
+				}
+				fmt.Fprintln(w, tierName)
+				fmt.Fprintf(w, "  Total size: %s\n", humanize.Bytes(tierUsage.Total))
+				fmt.Fprintf(w, "  Free: %s\n", humanize.Bytes(tierUsage.Free))
```

Comment:
If this is the same as used by pool query, can we break it out into a helper function that can be used by both?

Reply:
It is slightly different, with a different type here (StorageTargetUsage) having just total, free, versus pool query StorageUsageStats with min, max, mean, total, and free.

Reply:
Maybe we could share the tier switch statement though? My thought is we probably want this formatted text to stay the same for both.

Reply:
got it. added a helper to generate the strings to be printed based on the index value.